README.md

```diff
@@ -110,6 +110,8 @@ sudo chmod +x /usr/local/bin/aurora-echo
   - Allows multiple inputs (use one option flag per input).
 - `-h, --minimum-age-hours`
   - If an existing managed instance has been created within the last `-h` hours, abort creation of a new instance. Defaults to 20.
+- `-png, --db-parameter-group-name'`
```

Comment:
This documentation doesn't match the short opt value of `-pgn` in code

Reply:
@whwright it's typo error. I will fix it.